### DISCUSSION TITLE
[Tailwind CSS React Spinner / Loader WebPage] [The site is not scrollable due to Incorrect inline css style]

### DISCUSSION THREAD

**Discussion by Sachin4219** [mdbootstrap/TW-Elements#2330] [Support from community]
Maybe only I am getting this error but This problem is with all the pages, i just first noticed it on the spinner page
The webpage for Tailwind CSS React Spinner / Loader has inline css style on body set to overflow:hidden which causes the webpage to not be scrollable. Although when the page first loads the scroll bar is visible until we scroll down to an example or Tailwind socials and referral component then it disappears.

Below are the error images in inspect mode
![image](https://github.com/mdbootstrap/TW-Elements-React/assets/91952010/9d76956e-31c3-4744-abbf-cca93456e641)
![image](https://github.com/mdbootstrap/TW-Elements-React/assets/91952010/41cfe8c2-1bfe-44e9-a6b7-918a4b8e36af)

**Comment by iprzybysz** [comment]
Hi @Sachin4219, we can't reproduce this issue - it works fine. Check if this problem also exists on incognito mode or other browsers.
